Makefile

```diff
@@ -0,0 +1,7 @@
+PROJ_VERSION = 6.3.0
+
+install_proj:
+	sudo apt update && sudo apt install -y wget build-essential pkg-config sqlite3 libsqlite3-dev clang
+	wget https://github.com/OSGeo/proj.4/releases/download/$(PROJ_VERSION)/proj-$(PROJ_VERSION).tar.gz && tar -xzvf proj-$(PROJ_VERSION).tar.gz
+	cd proj-$(PROJ_VERSION) && ./configure --prefix=/usr && make && sudo make install
+	rm -rf proj-$(PROJ_VERSION) proj-$(PROJ_VERSION).tar.gz
```

Comment:
both minor (as you wish):
I would remove this line.
Also replace all `&&` with newlines and comments like the ones of README previously.

Reply:
Il leave it like that. I don't think it's necessary to comment theses lines